includes/class-wcmp-export.php

```diff
@@ -1558,13 +1558,13 @@ public function is_postnl_destination($country_code) {
     }
 
     public function is_eu_country($country_code) {
-        $euro_countries = array('AT','BE','BG','CZ','DK','EE','FI','FR','DE','GB','GR','HU','IE','IT','LV','LT','LU','PL','PT','RO','SK','SI','ES','SE','MC','AL','AD','BA','IC','FO','GI','GL','GG','JE','HR','LI','MK','MD','ME','UA','SM','RS','VA','BY');
```

Comment:
Er moet nog gevraagd worden of dit mogelijk is voor de PostNL klanten.
Het kan namelijk gebeuren dat de kanten niet meer kunnen exporteren omdat het PostNL account daarvoor niet ingesteld zijn.

Reply:
done now (set GB back to EU)